plugins/lambda/project.json

```diff
@@ -51,7 +51,7 @@
       "executor": "@jscutlery/semver:version",
       "options": {
         "preset": "conventionalcommits",
-        "postTargets": ["github", "deploy"]
+        "postTargets": ["github", "build", "deploy"]
```

Comment:
Since `ngx-deploy-npm` is dropping their internal build to use `dependsOn` Nx Feature and `postTarget` will not execute the target's dependencies, we need to specify ngx-deploy-npm dependencies here.

See more info on [Semver's Documentation](https://github.com/jscutlery/semver?tab=readme-ov-file#triggering-executors-post-release)